src/components/pages/Order/Main/AdminPanel/AdminPanel.js

```diff
@@ -0,0 +1,93 @@
+import React, { useContext, useState } from "react"
+import styled from "styled-components"
+import AdminContext from "../../../../../context/AdminContext"
+import PanelTabButton from "./PanelTabButton"
+import { FaPen, FaPlus, FaChevronDown, FaChevronUp } from "react-icons/fa"
+import { theme } from "../../../../../theme"
+import PanelOption from "./Panel"
```

Comment:
```suggestion
import Panel from "./Panel"
```

il vaut mieux garder le même nom ici